cmd/subnetcmd/create.go

```diff
@@ -14,11 +14,12 @@ import (
 )
 
 var (
-	forceCreate  bool
-	useSubnetEvm bool
-	genesisFile  string
-	vmFile       string
-	useCustom    bool
+	forceCreate      bool
+	useSubnetEvm     bool
+	genesisFile      string
+	vmFile           string
+	useCustom        bool
+	subnetEvmVersion string
```

Comment:
I'm a little confused as to what you're asking since these aren't constants. What I'm doing here is setting up flags to assist with VM creation. 

I think we'll probably add a flag for each VM template we add. All but custom will take a version.

So we should have `--subnet-evm 1.2.2`, `--spaces 0.0.3`, `--blob 0.1.2`, `--custom`, etc.